packages/account/src/Sections/Profile/PhoneVerification/phone-verification-page.tsx

```diff
@@ -6,12 +6,15 @@ import { Localize } from '@deriv/translations';
 import ConfirmPhoneNumber from './confirm-phone-number';
 import { useHistory } from 'react-router';
 import { routes } from '@deriv/shared';
+import ConfirmYourEmail from './confirm-your-email';
 
 const PhoneVerificationPage = () => {
+    const [show_email_verification, should_show_email_verification] = React.useState(true);
```

Comment:
Why using state here? when you are not changing it's value.

Reply:
currently using it here because we have no API calls for phone-verification yet. This is temporary just to show the email_verification and phone verification page

Reply:
okay...